OWNERS

```diff
@@ -1,8 +1,8 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
-- wongma7
-- jsafrane
+- kubernetes-csi-approvers
```

Comment:
Please add wongma7.

Reply:
The problem here is kubernetes-csi-reviewers and approvers are not a member under the kubernetes-sigs.

We have two options:
1. Copy [KUBERNETES_CSI_OWNERS_ALIASES](https://github.com/kubernetes-csi/csi-release-tools/blob/master/KUBERNETES_CSI_OWNERS_ALIASES) list over
2. Do something similar to[ kustomize OWNERS](https://github.com/kubernetes-sigs/kustomize/blob/master/OWNERS), get kubernetes-csi-reviewers and approvers as a member of kubernetes-sigs

Reply:
Since kubernetes-csi alias does not work under kubernetes-sigs, can you just add the approver names directly?
- msau42
- saad-ali
- xing-yang